server/env_variables.yaml

```diff
@@ -27,11 +27,9 @@ env_variables:
   #
   ORIGIN: "<YOUR-ORIGIN>"
 
-  # Uncomment the USE_OIDC line to turn on OIDC authentication.
-  # Note that the value ov USE_OIDC is not important; any value will turn on
-  # OIDC authentication.
+  # ENVIRONMENT can be 'development' or 'production'.
   #
-  #USE_OIDC: "true"
+  ENVIRONMENT: "production"
```

Comment:
I'm not sure I yet understand how this file is consumed.  That colors the following comment.

If both production and development deployments are always going through terraform, what does this do for personal deployments, is it required, and should it ever be "production"?

Reply:
This file is included in app.yaml.

This is an open source repo and many people may choose to deploy it, following the instructions in README.md. I, for example, have two google cloud projects, one of which I consider "production" and the other "development" and neither of those go through terraform.

In PR #118, you added ENVIRONMENT to constants.py with the comment "Expects to be 'development' or 'production'". However, if someone is following the README.md instructions, constants.ENVIRONMENT will be None, and that comment is then incorrect.